edge/pkg/edged/edged_pods.go

```diff
@@ -993,32 +1020,31 @@ func (e *edged) updatePodStatus(pod *v1.Pod) error {
 	return err
 }
 
-func toKubeContainerStatus(phase v1.PodPhase, status *kubecontainer.ContainerStatus) v1.ContainerStatus {
+func toKubeContainerStatus(phase v1.PodPhase, status *v1.ContainerStatus) v1.ContainerStatus {
```

Comment:
```suggestion
func toKubeContainerStatus(phase v1.PodPhase, status *kubecontainer.Status) v1.ContainerStatus {
```

Reply:
There's a ID here. https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/container/runtime.go#L310

If it's necessarily to refactor, please do it in follow up PRs. It's better not to make changes here.